user/utility.cpp

```diff
@@ -563,7 +564,7 @@ std::string ToString(__maybenull PlayerControl* player) {
 std::string ToString(__maybenull GameData_PlayerInfo* data) {
 	if (data) {
 		if (const auto outfit = GetPlayerOutfit(data))
-			return std::format("<#{} {}>", +data->fields.PlayerId, convert_from_string(outfit->fields._playerName));
+			return std::format("<#{} {}>", +data->fields.PlayerId, convert_from_string(outfit->fields.postCensorName));
```

Comment:
Is there an alternative to postCensorName? Would using it here mess with places where we change the player name (like shapeshifting or impersonation)?
Same with all the other usages of postCensorName in this file.

Reply:
A better way is to call GameData_PlayerOutfit::get_PlayerName(), but it's in nested class, get_method() can't resolve the nested class yet.